modules/unsupported/geojsonstore/src/main/java/org/geotools/data/geojson/GeoJSONReader.java

```diff
@@ -128,6 +129,44 @@ public static SimpleFeature parseFeature(String json) throws JsonParseException,
         }
     }
 
+    /**
+     * @param jsonString
+     * @return
+     */
+    public static SimpleFeatureCollection parseFeatureCollection(String jsonString) {
+
+        try (GeoJSONReader reader =
+                new GeoJSONReader(new ByteArrayInputStream(jsonString.getBytes()))) {
+            SimpleFeatureCollection features = (SimpleFeatureCollection) reader.getFeatures();
+            return features;
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
Nope? :-D

I would have thought we'd have a QA rule against this, but nope, it's not there:
https://github.com/geotools/geotools/blob/master/build/qa/pmd-ruleset.xml
It could, however, be added. Will try to remember about it (not asking you to do it and fix all issues, you have enough on your plate with this geojson switch):
https://pmd.github.io/pmd-6.31.0/pmd_rules_java_bestpractices.html#avoidprintstacktrace